week-2/Homework/mandatory/3-playing-computer.js

```diff
@@ -7,36 +7,36 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
-  2. Remove the line that throws the error.
+  2. Remove the line that throws the error. 
   3. What is printed to the console?
```

Comment:
Your comments of what's happening in each part of the file are good :) Can you also add the answers to these questions to explain how the whole program behaves?